tests/TestCase/Model/Filter/ValueTest.php

```diff
@@ -46,6 +46,25 @@ public function testSkipProcess()
     /**
      * @return void
      */
+    public function testProcessWithoutField()
+    {
+        $articles = TableRegistry::get('Articles');
+        $manager = new Manager($articles);
+        $filter = new Value('title', $manager, [
+            'field' => []
```

Comment:
That's a good point. Initially I've added it for the other filters in the first round of the multi-field support changes, in order to avoid invalid queries being generated, but since I forgot to add specific tests for that behavior, I now don't know the scenario anymore in which I saw this happening :(

I'd agree that throwing an exception would probably the appropriate thing to do, however that would be a BC break at this point I guess?